### ISSUE TITLE
Triage broken E2E tests

### ISSUE THREAD

**Issue by NoopDog** [DataBiosphere/data-browser#4107]
https://github.com/DataBiosphere/data-browser/actions/runs/10373869841

Is this an actual failure or a test failure?
What neds to be done to fix.

**Comment by frano-m** [comment]
Currently, these tests appear to be failing:

```
anvil/anvil-pagination.spec.ts:36:5 › Check that forward pagination increments the current page and that page count stays static for the first five pages on the donors tab 
anvil/anvil-sort.spec.ts:7:5 › Expect clicking each column header three times to keep the first text element visible on the Datasets tab 
anvil/anvil-sort.spec.ts:31:5 › Expect clicking each column header three times to keep the first text element visible on the Files tab 
anvil/anvil-tabs-buttons.spec.ts:37:5 › Expect clicking the biosamples tab to go to the correct url and to show all of the relevant columns when selected 
```

**Comment by mrtopsyt** [comment]
This was due to an indexing issue, will be resolved by #4108
